.eslintrc.js

```diff
@@ -27,5 +27,18 @@ module.exports = {
   ],
 
   rules: {},
-  overrides: [],
+  overrides: [
+    {
+      "files": ["*cli.ts"],
+      "rules": {
+        "node/shebang": "off"
+      },
+    },
+    {
+      "files": ["src/commands/*.ts"],
+      "rules": {
+        "class-methods-use-this": "off",
+        "@typescript-eslint/no-unsafe-assignment": "off"
```

Comment:
I disabled class-methods-use-this because all the command classes extend the abstract Command class and have to implement 2 methods (command and description) that simply describe the concrete command. As such, they don't use `this`.  The warning was popping up all over the place.

I'll remove the override for no-unsafe-assignment. There's just one class that needs that disabled.

Reply:
Removed no-unsafe-assignment rule.